packages/components/src/components/hds/form/toggle/field.ts

```diff
@@ -0,0 +1,31 @@
+import templateOnlyComponent from '@ember/component/template-only';
+import type { HdsFormFieldSignature } from '../field';
+import type { ComponentLike } from '@glint/template';
+import type { HdsFormLabelSignature } from '../label';
+import type { HdsFormHelperTextSignature } from '../helper-text';
+import type { HdsFormErrorSignature } from '../error';
+
+export interface HdsFormToggleFieldSignature {
+  Args: {
```

Comment:
Some of these are passed down to `Hds::Form::Field`, should we link these args?

Reply:
Addressed in f5dbe5b